test/syncenginetestutils.h

```diff
@@ -150,6 +153,7 @@ class FileInfo : public FileModifier
     void modifyLockState(const QString &relativePath, LockState lockState, int lockType, const QString &lockOwner, const QString &lockOwnerId, const QString &lockEditorId, quint64 lockTime, quint64 lockTimeout) override;
 
     void setE2EE(const QString &relativepath, const bool enabled) override;
+    void setFolderQuota(const QString &relativePath, const FolderQuota newQuota);
```

Comment:
similarly to the `find`method, can you add a specific parameter to indicate that etag is going to be modified or not modified ?
for example: `const bool invalidateEtags = false` (or even better with an enum that avoids the `bool` API hell)